### ISSUE TITLE
Cannot crop Screenshots on iOS 11

### ISSUE THREAD

**Issue by JerryAndrews** [ytakzk/Fusuma#218]
Hello Everyone,

This is my first time posting on GitHub, I did not know if I should post this finding as an addendum to a current issue or post this as a new issue altogether. I did both in case one was not the correct means to discuss this.

From my testings on iPhone 7 Plus running iOS 11:

When selecting a vertical or horizontal screenshot, that has not been modified, fusuma does not crop the image. It returns the full horizontal or vertical screenshot.

It appears @j-down had a similar issue when selecting screenshots specifically.

It appears to me that the issue is when the width/height of the image is equivalent to the device width/height fusuma is not able to crop. Hence why it does not crop with screenshots.

To test this theory:

I took one of the horizontal screenshots from my device, and cropped a minute portion of the width. When I proceeded to crop in fusuma I got the desired image.

I took one of the vertical screenshots from my device, and cropped a minute portion of the height. When I proceeded to crop in fusuma I got the desired image.

I tested with an image of dimensions 5500 × 3637. I was able to crop without issue.

I tested with a picture taken from the front facing cam. I was able to crop without issue.

I tested with a picture taken from the rear facing cam. I was able to crop without issue.

In summary the issue is only happening with device screenshots.

Thank you.

**Comment by ytakzk** [comment]
#215
